benchmark/distributed_nonhydrostatic_model_mpi.jl

```diff
@@ -11,8 +11,6 @@ using Benchmarks
 
 Logging.global_logger(OceananigansLogger())
 
-MPI.Init()
-
```

Comment:
I think this is necessary --- or at least I can't run this script without it, because I get something like

```julia
[2022/03/13 13:00:26.020] INFO  Benchmarking weak scaling nonhydrostatic model with Slab decomposition [N=(128, 128, 16), ranks=(1, 1, 1)]...
Invalid MIT-MAGIC-COOKIE-1 keyInvalid MIT-MAGIC-COOKIE-1 keyNo protocol specified
[ Info: Oceananigans will use 24 threads
┌ Warning: Package Benchmarks does not have Oceananigans in its dependencies:
│ - If you have Benchmarks checked out for development and have
│   added Oceananigans as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with Benchmarks
└ Loading Oceananigans into Benchmarks from project dependency, future warnings for Benchmarks are suppressed.
[ Info: Oceananigans will use 24 threads
*** The MPI_Comm_rank() function was called before MPI_INIT was invoked.
*** This is disallowed by the MPI standard.
*** Your MPI job will now abort.
```

Reply:
Thanks for catching that - Init_MPI() should definitely stay!

Reply:
Ok, I put it back in, hope that's ok!